Adafruit_ADT7410.cpp

```diff
@@ -37,24 +37,71 @@
  */
 Adafruit_ADT7410::Adafruit_ADT7410() {}
 
+/*!
+ *    @brief  Setups the HW with default address and Wire object
+ *    @return True if initialization was successful, otherwise false.
+ */
+bool Adafruit_ADT7410::begin() {
+  _i2caddr = ADT7410_I2CADDR_DEFAULT;
+  _wire = &Wire;
+  return init();
+}
+
+/*!
+ *    @brief  Setups the HW with default address
+ *    @param  *wire
+ *    @return True if initialization was successful, otherwise false.
+ */
+bool Adafruit_ADT7410::begin(TwoWire *wire) {
+  _i2caddr = ADT7410_I2CADDR_DEFAULT;
+  _wire = wire;
+  return init();
+}
+
+/*!
+ *    @brief  Setups the HW with default Wire object
+ *    @param  addr
+ *    @return True if initialization was successful, otherwise false.
+ */
+bool Adafruit_ADT7410::begin(uint8_t addr) {
+  _i2caddr = addr;
+  _wire = &Wire;
+  return init();
+}
+
```

Comment:
these other `begin()`s can call the "full" version of `begin()` and just substitute the parameters as needed for the particular case. Ex:
```cpp
bool Adafruit_ADT7410::begin() {
  return begin(ADT7410_I2CADDR_DEFAULT, &Wire);
}
bool Adafruit_ADT7410::begin(TwoWire *wire) {
  return begin(ADT7410_I2CADDR_DEFAULT, wire);
}
bool Adafruit_ADT7410::begin(uint8_t addr) {
  return begin(addr, &Wire);
}
```